### PR TITLE
set up unit tests

### PR DESCRIPTION
**Pull request by Rufai-Alex**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [ ] I have run the app using my feature and ensured that no functionality is broken
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by walela** [comment]
[Set up unit tests](https://trello.com/c/wsdCCaRv/39-set-up-unit-tests)

File: .travis.yml [hunk 1]
```diff
@@ -6,15 +6,17 @@ cache:
...
     - 'input'
 before_script:
   - npm install make-coverage-badge
 script:
-  - npm test
+  - npm run test
 after_script:
   - make-coverage-badge --output-path ./badges
+  
+
```
Comment (Ofega) [inline .travis.yml]: Please run the format script on this to make sure the code is well formatted

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.activeBackground": "#2f7c47",
+    "activityBar.activeBorder": "#422c74",
+    "activityBar.background": "#2f7c47",
```
Comment (Ofega) [inline .vscode/settings.json:5]: I'm not sure why this is on here, maybe add this file to your .env

Comment (walela) [reply .vscode/settings.json:5]: sorry, that is my local vs code settings file. maybe I need to add it to the .gitignore

**Pull request merged**